2003/mtjin/AndroidArchitectureStudy/app/src/main/java/com/mtjin/androidarchitecturestudy/ui/LoginActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.mtjin.androidarchitecturestudy.ui
+
+import android.content.Intent
+import android.os.Bundle
+import android.widget.Button
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.mtjin.androidarchitecturestudy.R
+import com.mtjin.androidarchitecturestudy.utils.PreferenceManager
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var tvId: TextView
+    private lateinit var tvPw: TextView
+    private lateinit var btnLogin: Button
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_login)
+        initView()
+        initListener()
+    }
+
+    private fun initView() {
+        tvId = findViewById(R.id.tv_id)
+        tvPw = findViewById(R.id.tv_pw)
+        btnLogin = findViewById(R.id.btn_login)
+    }
+
+    private fun initListener() {
+        btnLogin.setOnClickListener {
+            val id = tvId.text.toString().trim()
+            val pw = tvPw.text.toString().trim()
+            if (id == USER_ID && pw == USER_PW) {
+                startActivity(Intent(this, MovieSearchActivity::class.java))
+                PreferenceManager.setBoolean(this, PreferenceManager.AUTO_LOGIN_KEY, true)
```

Comment:
로그인이 완료된 후 뒤로가기를 누르면 어떻게 될까요?
```suggestion
                PreferenceManager.setBoolean(this, PreferenceManager.AUTO_LOGIN_KEY, true)
                startActivity(Intent(this, MovieSearchActivity::class.java))
                finish()
               
```

Reply:
수정했습니다.

ed62341